.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max_line_length=120
+ignore=E305,W504
```

Comment:
E305 is https://www.flake8rules.com/rules/E305.html and w504 is https://www.flake8rules.com/rules/W504.html

Reply:
Okay